lib/cli/index.js

```diff
@@ -30,22 +36,14 @@ const handleFile = ({ write, verbose }, config) => async filePath => {
 
 async function execute(argv) {
   try {
-    const path = require('path');
-
-    const Timer = require('../utils/timer');
     const timer = new Timer();
     timer.start();
 
-    const {
-      globs,
-      config: configPath,
-      ignore,
-      ...options
-    } = require('./parse')(argv);
+    const { globs, config: configPath, ignore, ...options } = parser(argv);
 
-    const config = configPath ? require(configPath) : {};
+    const { config } = await configSearch.search({ configPath });
```

Comment:
We should likely be handling the possible `error` property value being set from this call

Reply:
@camacho the error is handled by the `.catch` in the `search()` in `config.js`. The current behavior is to fallback to the default configuration in `lib/defaults/index.js`. In a different PR I would like to rework the way how we do logging (e.g `--loglevel warn`).

Reply:
The second part is more what I was curious about. Improving logging is a good idea.